pkg/controller/constrainttemplate/constrainttemplate_controller.go

```diff
@@ -316,7 +311,6 @@ func (r *ReconcileConstraintTemplate) Reconcile(request reconcile.Request) (reco
 	}
 	unversionedProposedCRD, err := r.opa.CreateCRD(context.Background(), unversionedCT)
 	if err != nil {
-		r.tracker.CancelTemplate(unversionedCT) // Don't track templates that failed compilation
```

Comment:
IIRC calling CancelTemplate is important because it propagates the cancellation to all of the constraints that were expected to appear as well.